src/wcs2_0/src/test/java/org/geoserver/wcs2_0/response/WCS20GetCoverageResponseTest.java

```diff
@@ -0,0 +1,30 @@
+/* (c) 2021 Open Source Geospatial Foundation - all rights reserved
```

Comment:
Looks to be related, but a different issue being addressed?

Reply:
Indeed it's part of its own commit which refers to its own jia ticket: "[GEOS-10156] WCS20GetCoverageResponse does not advertise the list of supported MIME types"

Reply:
My bad - missed the second commit.